### DISCUSSION TITLE
Not pulling down 13.4.1c?

### DISCUSSION THREAD

**Discussion by g003441** [grahampugh/erase-install#420] [General]
I noticed when running "/Library/Management/Erase-install/erase-install.sh --reinstall --os 13 --current-user" it pulls down 13.4.1 and not the latest 13.4.1c. Is this an erase-install limitation or am I doing something wrong?

**Comment by grahampugh** [comment]
Apple do not roll in Rapid Security Responses into full installers, but erase-install is a tool for installing full installers. So I guess you're wishing for something beyond what erase-install can do.

**Reply by g003441** [reply]
Sounds like! Thank you.
